### PR TITLE
Sign macos and release ARM executables for macos.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -566,11 +591,24 @@ jobs:
 
       - uses: actions/download-artifact@v3
         with:
-          name: macOS-build
+          name: macOS-build-unsigned
+
+      - name: macOS arm environment
```
Comment (kasperl) [inline .github/workflows/ci.yml:596]: Typically these names have more of a verb'y sound to them like "Do X". What does this do?

```suggestion
      - name: macOS ARM environment
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -816,6 +851,83 @@ jobs:
...
+          EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
+          echo "result<<$EOF" >> $GITHUB_OUTPUT
+          # Don't sign the esptool which isn't from us.
+          find toit-x64/bin toit-x64/tools toit-arm/bin toit-arm/tools \
+             -type f \
+             ! -regex '.*/esptool' \
+             >> $GITHUB_OUTPUT
+          echo "$EOF" >> $GITHUB_OUTPUT
+
+      - name: Sign macOS binaries
```
Comment (kasperl) [inline .github/workflows/ci.yml:885]:
```suggestion
      - name: Sign macOS executables
```

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (floitsch) [reply .github/workflows/ci.yml:596]: done.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (floitsch) [reply .github/workflows/ci.yml:885]: done.

**Pull request merged**